docs/java.md

```diff
@@ -0,0 +1,373 @@
+            .withAuthProvider(new PlainTextAuthProvider("scylla", "*******"))  
+            .build();  
+            
+            Session session = cluster.connect();  
+
+            PreparedStatement statement = session.prepare(
+                "INSERT INTO media_player.songs (id, title, artist, album, created_at, updated_at) VALUES (?,?,?,?,?,?)"
+            );
+
+            BoundStatement bound = statement.bind()
+                    .setLong(0, 1)
+                    .setString(1, "Stairway to Heaven")
+                    .setString(2, "Led Zeppelin")
+                    .setString(3, "Led Zeppelin IV")
+                    .setTimestamp(4, new Date())
+                    .setTimestamp(5, new Date());
+
+            session.execute(bound);
+     }
+}
+```
+
+### 3.3 Reading data
+
+Since probably we added more than 3 songs into our database, let's list it into our terminal.
+
+```java
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.BoundStatement;
+import com.datastax.driver.core.PreparedStatement;
+import com.datastax.driver.core.Row;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.core.PlainTextAuthProvider;
+import com.datastax.driver.core.policies.DCAwareRoundRobinPolicy;
+
+import java.util.Date;
+import java.util.Scanner;
+
+class Main {  
+  
+    public static void main(String[] args) { 
+        Cluster cluster = Cluster.builder()  
+            .addContactPoints("your-node-url.7207ca5a8fdc45f2b03f.clusters.scylla.cloud", "your-node-url.7207ca5a8fdc45f2b03f.clusters.scylla.cloud", "your-node-url.7207ca5a8fdc45f2b03f.clusters.scylla.cloud")  
+            .withLoadBalancingPolicy(DCAwareRoundRobinPolicy.builder().withLocalDc("AWS_US_EAST_1").build()) // your local data center  
+            .withAuthProvider(new PlainTextAuthProvider("scylla", "*******"))  
+            .build();  
+            
+            Session session = cluster.connect();  
+
+            ResultSet songsResult = session.execute("SELECT * FROM media_player.songs PER PARTITION LIMIT 1");
+            List<Row> songList = results.all();
+
+            for (Row row : resultsmemo) {
+                System.out.println(
+                    String.format(
+                            "ID: %d -> Song: %s -> Artist: %s -> Album: %s -> Created At: %s",
+                            row.getInt("id"),
+                            row.getString("title"),
+                            row.getString("artist"),
+                            row.getString("album"),
+                            row.getTimestamp("created_at").toString()
+                    )
+            );
+        }
+     }
+}
+```
+
+
+### 3.4 Updating Data
+
+Ok, almost there! Now we're going to learn about `UPDATE` but here's a disclaimer: 
+> INSERT and UPDATES are not the same!
+
+There's a myth in Scylla/Cassandra community that it's the same for the fact that you just need the `Partition Key` and `Clustering Key` (if you have one) and query it.
```

Comment:
I think UPDATE and INSERT are the same (or very similar)
If we claim its not, we need to explain why.

Reply:
INSERT and UPDATE are almost identical. The only difference is the so-called _cql row marker_ that is added by INSERT but not by UPDATE. I think the information is not necessary here and should be removed.

Reply:
Removed.